bundles/org.openhab.binding.network/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -54,6 +54,14 @@
 				<default>5000</default>
 				<advanced>true</advanced>
 			</parameter>
+
+			<parameter name="networkInterfaceNames" type="text" required="false" multiple="true">
```

Comment:
Please also update README to add the description of the two new parameters.

Reply:
I have added a commit for this!